pep/app/pods/user/model.ts

```diff
@@ -24,19 +26,60 @@ interface ActiveSubscription {
 export default class User extends DS.Model {
     @service intl!: IntlService;
 
-    @attr('string') activeSubscriptions!: string;
-    @attr('boolean') branding!: boolean;
-    @attr('string') brandingImgUrl!: string;
-    @attr<any>('json', { defaultValue: () => ({ ...DEFAULT_USER_PREFERENCES }) }) clientSettings!: UserPreferences;
-    @attr('string') emailAddress!: string;
-    @attr('boolean') hasArchiveAccess!: boolean;
-    @attr('boolean') hasCurrentAccess!: boolean;
-    @attr('string') loggedInMethod!: string;
-    @attr('string') paDSHomeURL!: string;
-    @attr('date') subscriptionEndDate!: Date;
-    @attr('string') userFullName!: string;
-    @attr('string') userName!: string;
-    @attr('string') userType!: UserType;
+    @alwaysSend
+    @attr('string')
+    activeSubscriptions!: string;
+    @alwaysSend
+    @attr('boolean')
+    branding!: boolean;
+    @alwaysSend
+    @attr('string')
+    brandingImgUrl!: string;
+    @alwaysSend
+    @attr<any>('json', { defaultValue: () => ({ ...DEFAULT_USER_PREFERENCES }) })
+    clientSettings!: UserPreferences;
+    @alwaysSend
+    @attr('string')
+    emailAddress!: string;
+    @alwaysSend
+    @attr('boolean')
+    hasArchiveAccess!: boolean;
+    @alwaysSend
+    @attr('boolean')
+    hasCurrentAccess!: boolean;
+    @alwaysSend
+    @attr('boolean')
+    isSubscriptionRenewalDue!: boolean;
+    @alwaysSend
+    @attr('string')
+    loggedInMethod!: string;
+    @alwaysSend
+    @attr('string')
+    paDSHomeURL!: string;
+    @alwaysSend
+    @attr('string')
+    paDSRenewalURL!: string;
+    @alwaysSend
+    @attr('string')
+    reasonDescription!: string;
+    @alwaysSend
+    @attr('boolean')
+    sendJournalAlerts!: boolean;
+    @alwaysSend
+    @attr('boolean')
+    sendVideoAlerts!: boolean;
+    @alwaysSend
+    @attr('date')
+    subscriptionEndDate!: Date;
+    @alwaysSend
+    @attr('string')
+    userFullName!: string;
+    @alwaysSend
+    @attr('string')
+    userName!: string;
+    @alwaysSend
+    @attr('string')
+    userType!: UserType;
```

Comment:
this is alot harder to scan down the list to read model attributes now, imo. if we dont want to keep the single line decorator format here, e.g. `@alwaysSend @attr('string') userName!: string;`, i think we should at least have an empty line between each attribute when using newline-separated decorator format.

Reply:
This is how `prettier` automatically formats it. I can add the empty line though

Reply:
not very pretty, prettier. 😤